backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/DefaultIOSInput.java

```diff
@@ -799,8 +800,13 @@ private void toTouchEvents (long touches) {
 			// Get and map the location to our drawing space
 			{
 				CGPoint loc = touch.getLocationInView(app.graphics.view);
-				locX = (int)(loc.getX() - screenBounds.x);
-				locY = (int)(loc.getY() - screenBounds.y);
+				if (config.hdpiMode == HdpiMode.Pixels) {
```

Comment:
I think it is better understandable what's going on if we would have code like this:

			locX = (int) (loc.getX() - screenBounds.x);
			locY = (int) (loc.getY() - screenBounds.y);
			if (config.hdpiMode == HdpiMode.Pixels) {
				locX = locX * app.pixelsPerPoint;
				locY = locY * app.pixelsPerPoint;
			}

Reply:
Sure, updated.